arviz/plots/posteriorplot.py

```diff
@@ -42,16 +42,16 @@ def plot_posterior(
     Parameters
     ----------
     data: obj
-        Any object that can be converted to an az.InferenceData object
-        Refer to documentation of az.convert_to_dataset for details
+        Any object that can be converted to an :class:`arviz.InferenceData` object
+        refer to documentation of :func:`arviz.convert_to_dataset` for details
```

Comment:
```suggestion
        Refer to documentation of :func:`arviz.convert_to_dataset` for details
```

Reply:
Wait I did make all these changes, but they aren't reflecting here, should I try committing again?

Reply:
Solved this